M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -1274,24 +1348,31 @@ doc ///
     Key
         chooseGoodMinors
         (chooseGoodMinors, ZZ, ZZ, Matrix)
+        (chooseGoodMinors, ZZ, ZZ, Matrix, Ideal)
         [chooseGoodMinors, Verbose]
         [chooseGoodMinors, PeriodicCheckFunction]
         PeriodicCheckFunction
     Headline
         returns an ideal generated by interesting minors in a matrix
     Usage
         chooseGoodMinors(count, minorSize, M)
+        chooseGoodMinors(count, minorSize, M, I)
     Inputs
         count: ZZ
         minorSize: ZZ
         M: Matrix
+        I: Ideal            
     Outputs
         : Ideal
     Description
         Text
-            This returns an ideal generated by approximately {\tt count} minors of size {\tt minorSize} the matrix M.
+            This returns an ideal generated by approximately {\tt count} minors of size {\tt minorSize} the matrix M.  
         Example
             R = QQ[x, y, z];
+            M = matrix{{x,y,0}, {y,z,0}, {0,0,0}}, 
+            chooseGoodMinors(1, 2, M, Strategy=>StrategyDefaultNonRandom)
+        Text
+            If the ideal is passed, then it is used if the when the {\tt Points} portion of the {\tt Strategy} is turned on.  
```

Comment:
"if the when the" ??

Explain how the ideal is used.

Reply:
I've clarified this, to be pushed shortly.